src/imagery/i.eodag/i.eodag.py

```diff
@@ -37,66 +35,83 @@
 # % answer: S1_SAR_GRD
 # % guisection: Filter
 # %end
-
+# %option G_OPT_V_MAP
+# % description: If not given then current computational extent is used
+# % label: Name of input vector map to define Area of Interest (AOI)
+# % required: no
+# % guisection: Region
+# %end
+# %option
+# % key: clouds
+# % type: integer
+# % description: Maximum cloud cover percentage for Sentinel scene
+# % required: no
+# % guisection: Filter
+# %end
+# %option G_OPT_V_OUTPUT
+# % key: footprints
+# % description: Name for output vector map with footprints
+# % label: Only supported for download from ESA_Copernicus Open Access Hub
```

Comment:
I guess this is a leftover from i.sentinel, right?

Reply:
I got it from there yes, for when I add the footprint saving feature. Didn't implement it yet.